auto_round/script/mllm.py

```diff
@@ -269,6 +269,8 @@ def tune(args):
             args.device = ",".join(map(str, range(len(devices))))
             devices = args.device.replace(" ", "").split(',')
         use_auto_mapping = True
+    elif args.device == "auto":
```

Comment:
change this in llm too.  llm may need to change two place, tuning and evaluation

Reply:
besides, if you want to use `auto` to use multigpu,  it's not the expected behavior in the main code.  The gpu device number is controlled by --devices now